music21/braille/segment.py

```diff
@@ -1661,20 +1650,8 @@ def prepareSlurredNotes(music21Part, **keywords):
     '''
     if not music21Part.spannerBundle:
         return
-
-    slurLongPhraseWithBrackets = keywords.get('slurLongPhraseWithBrackets',
-                                                  SEGMENT_SLURLONGPHRASEWITHBRACKETS)
-    showShortSlursAndTiesTogether = keywords.get('showShortSlursAndTiesTogether',
-                                                     SEGMENT_SHOWSHORTSLURSANDTIESTOGETHER)
-    if 'showLongSlursAndTiesTogether' in keywords:
-        showLongSlursAndTiesTogether = keywords['showLongSlursAndTiesTogether']
-    elif slurLongPhraseWithBrackets:
-        showLongSlursAndTiesTogether = True
-    else:
-        showLongSlursAndTiesTogether = SEGMENT_SHOWLONGSLURSANDTIESTOGETHER
-
-    if slurLongPhraseWithBrackets is False:
-        pass
+    if not slurLongPhraseWithBrackets:
+        showLongSlursAndTiesTogether = False
```

Comment:
FYI -- I'd like another pair of eyes on this. This was deeply confusing, and not apparent how it could be recreated using method signatures.

The issue is the two keywords `showLong...` and `slurLong...`

Previously, if `showLong...` was silent, we just allied `slurLong` and `showLong` to the same value. `slurLong` defaulted True, so we got True and True. This is why I made the change to the documentation for the keyword `showLong`--it said the default was False, but that's not what was happening.

However, if you explicitly set `showLong`, you get that value, even if it conflicts with `slurLong`. I don't think this is tested anywhere.

To get the tests to pass, I had to keep the current behavior for showLong=silent and slurLong=False, but since there is no way to represent silence (without using Optional[bool]=None), I just did `if not slurLong: showLong=False`, which means there is no way to affirmatively provide `slurLong=True` and `showLong=False`, which was theoretically possible but not tested.

Acceptable? Or do you want to insert some sort of Optional[bool]=None for showLong? Given the pretzel here and the fact the documentation was misleading and no tests are failing, this seemed fair.

Reply:
♻️  'ing back to this, I think `Optional[bool]=None` is the most conservative thing we can do to ensure we don't break people's workflows. I'll spruce this up.